lib/readwise/client.rb

```diff
@@ -96,16 +110,55 @@ def get_export_page(page_cursor: nil, updated_after: nil, book_ids: [])
       params['updatedAfter'] = updated_after if updated_after
       params['ids'] = book_ids if book_ids.any?
       params['pageCursor'] = page_cursor if page_cursor
-      url = 'https://readwise.io/api/v2/export/?' + URI.encode_www_form(params)
-      export_uri = URI.parse(url)
-      export_req = Net::HTTP::Get.new(export_uri)
-      export_req['Authorization'] = "Token #{@token}"
-      export_res = Net::HTTP.start(export_uri.hostname, export_uri.port, use_ssl: true) do |http|
-        http.request(export_req)
+      url = BASE_URL + 'export/?' + URI.encode_www_form(params)
+
+      begin
+        res = make_readwise_request(url)
+      rescue => exception
+        raise Error, exception.message
       end
-      raise Error, 'Export request failed' unless export_res.is_a?(Net::HTTPSuccess)
 
-      JSON.parse(export_res.body)
+      res
+    end
+
+    def transform_highlight(res)
+      Highlight.new(
+        book_id: res['book_id'].to_s,
+        color: res['color'],
+        created_at: res['created_at'],
+        end_location: res['end_location'],
+        external_id: res['external_id'],
+        highlight_id: res['id'].to_s,
+        highlighted_at: res['highlighted_at'],
+        is_discard: res['is_discard'],
+        is_favorite: res['is_favorite'],
+        location: res['location'],
+        location_type: res['location_type'],
+        note: res['note'],
+        readwise_url: res['readwise_url'],
+        tags: res['tags'].map do |tag|
+          Tag.new(
+            tag_id: tag['id'].to_s,
+            name: tag['name']
+          )
+        end,
+        text: res['text'],
+        updated_at: res['updated_at'],
+        url: res['url'],
+      )
+    end
+
+    def make_readwise_request(url)
```

Comment:
```suggestion
    def get_readwise_request(url)
```

I would name this to signify that it's a `GET` request (because we'll want to support other methods in the future) - _or_ make the HTTP verb a parameter.